packages/nitro-rpc-client/src/serde.ts

```diff
@@ -98,14 +98,36 @@ const paymentSchema = {
 } as const;
 type PaymentSchemaType = JTDDataType<typeof paymentSchema>;
 
+const voucherSchema = {
+  properties: {
+    ChannelId: { type: "string" },
+    Amount: { type: "uint32" },
+    Signature: {
+      type: "string",
+    },
+  },
+} as const;
```

Comment:
Is it necessary to cast `as const`? This is already declared as a `const`...

Reply:
I don't know why, but it seems necessary.  If I don't add the `as const`(like the other entries) I get a type error lower down. I'm sure if I spent some time relearning the typescript type system I could tell you why but I don't think I'm willing to do that 😛